src/nodes/Paragraph.js

```diff
@@ -19,6 +19,20 @@ const Paragraph = TiptapParagraph.extend({
 
 	addKeyboardShortcuts() {
 		return {
+			Enter: ({ editor }) => {
+				// Fix for firefox
+				if (window.triggeredModBackspace) {
```

Comment:
This seems more like a dirty hack, could we rather please try to figure out the root cause? If it is not a problem in tiptap, there must be some difference in why our editor triggers a new line.